Dockerfile

```diff
@@ -1,46 +1,11 @@
-FROM openjdk:11-slim-buster AS builder
-COPY artifact/*.jar wdk.jar
-
-# Create custom JRE
-RUN mkdir custom
-RUN jlink --no-header-files --no-man-pages --compress=2 --strip-debug --add-modules java.base,java.scripting,java.sql,jdk.unsupported,java.se,jdk.crypto.ec --output custom/jre
+FROM openjdk:17-alpine
+RUN jlink --no-header-files --no-man-pages --compress=2 --strip-java-debug-attributes --output /jre \
```

Comment:
is jdeps necessary before jlink ?

Reply:
only if the deps will change due to new dependencies in the JAR. since wdk is quite stable now, a static list of deps in jlink should work. just need to verify that any major changes in the future doesn't break this.